lib/cluster/index.ts

```diff
@@ -910,15 +915,12 @@ const scanCommands = [
 scanCommands.forEach((command) => {
   Cluster.prototype[command + "Stream"] = function (key, options) {
     return new ScanStream(
-      defaults(
-        {
-          objectMode: true,
-          key: key,
-          redis: this,
-          command: command,
-        },
-        options
-      )
+      Object.assign({}, options, {
+        objectMode: true,
```

Comment:
This can cause a different result as `objectMode` will always be `true` regardless of `options`.

Reply:
So all good here?